chatkit-core/src/test/kotlin/com/pusher/chatkit/RoomStoreReceivingNewInitialStatesSpek.kt

```diff
@@ -0,0 +1,185 @@
+package com.pusher.chatkit
+
+import com.google.common.truth.Truth.assertThat
+import com.pusher.chatkit.memberships.MembershipSubscriptionEvent
+import com.pusher.chatkit.rooms.Room
+import com.pusher.chatkit.rooms.RoomStore
+import com.pusher.chatkit.users.ReadStateApiType
+import com.pusher.chatkit.users.User
+import com.pusher.chatkit.users.UserSubscriptionEvent
+import org.junit.Assert.fail
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class RoomStoreReceivingNewInitialStatesSpek : Spek({
```

Comment:
It's not a new test file. It's previous RoomStoreSpec with more specific name and fix to make the tests independent (memoized) and also making one test being actually a test (missing `it`). Full context: 7684bff8c49a7ade82bcbf602e2cbc3a48b50e1c